pallets/frequency-tx-payment/src/benchmarking.rs

```diff
@@ -38,6 +69,104 @@ mod benchmarks {
 		Ok(())
 	}
 
+	#[benchmark]
+	fn charge_tx_payment_free() {
```

Comment:
Benchmarks  full extension run for free, tokenized and capacity enabled txs and compute weights for each path